rure/__init__.py

```diff
@@ -9,8 +11,18 @@
     "compile", "search", "is_match", "match", "findall", "finditer",
     "DEFAULT_FLAGS",
     "CASEI", "MULTI", "DOTNL", "SWAP_GREED", "SPACE", "UNICODE",
+    "I", "IGNORECASE", "L", "LOCALE", "U", "M", "MULTILINE", "S",
+    "DOTALL", "X", "VERBOSE"
 ]
 
+# Make flag constants from stdlib re available here.
+I = IGNORECASE = CASEI
+L = LOCALE
+U = UNICODE
+M = MULTILINE = MULTI
+S = DOTALL = DOTNL
+X = VERBOSE = SPACE
+
```

Comment:
Yeah that's what I was going for was "ergonomics" (good word for it). You can close this if it's too far off the mark.